django/db/backends/base/base.py

```diff
@@ -214,10 +214,25 @@ def ensure_connection(self):
 
     # ##### Backend-specific wrappers for PEP-249 connection methods #####
 
-    def _cursor(self):
+    def _prepare_cursor(self, cursor):
+        """
+        Validate that connection is usable and perform database cursor
+        wrapping.
+        """
+        self.validate_thread_sharing()
+        if self.queries_logged:
+            wrapped_cursor = self.make_debug_cursor(cursor)
+        else:
+            wrapped_cursor = self.make_cursor(cursor)
+        return wrapped_cursor
+
+    def _cursor(self, name=None):
         self.ensure_connection()
         with self.wrap_database_errors:
-            return self.create_cursor()
+            if name:
```

Comment:
Did you consider updating the `def create_cursor` definition on the other backends so if/else isn't needed? This could be mentioned in the "Database backend API" changes in the release notes.

If we want to keep both signatures, something like `return self.create_cursor(*(name,) if name else ())` might be less repetitive. I see this pattern is used a few times.